cmd/create/execution.go

```diff
@@ -149,10 +149,12 @@ type ExecutionConfig struct {
 	Recover         string `json:"recover" pflag:",execution id to be recreated from the last known failure point."`
 	DryRun          bool   `json:"dryRun" pflag:",execute command without making any modifications."`
 	// Non plfag section is read from the execution config generated by get task/launchplan
-	Workflow string                 `json:"workflow,omitempty"`
-	Task     string                 `json:"task,omitempty"`
-	Version  string                 `json:"version"`
-	Inputs   map[string]interface{} `json:"inputs"`
+	Workflow    string                 `json:"workflow,omitempty"`
+	Task        string                 `json:"task,omitempty"`
+	Version     string                 `json:"version"`
+	Inputs      map[string]interface{} `json:"inputs"`
+	Labels      map[string]string      `json:"labels,omitempty"`
+	Annotations map[string]string      `json:"annotations,omitempty"`
```

Comment:
You will need to regenerate the pflags file for these new properties in the struct